src/rules/pycodestyle/rules.rs

```diff
@@ -78,6 +78,23 @@ pub fn line_too_long(lineno: usize, line: &str, settings: &Settings) -> Option<D
     }
 }
 
+/// E101
+pub fn mixed_spaces_and_tabs(lineno: usize, line: &str) -> Option<Diagnostic> {
+    let indent = leading_space(&line);
+
+    if indent.contains(" ") && indent.contains("\t") {
```

Comment:
Running `cargo clippy --workspace --all-targets --all-features -- -D warnings -W clippy::pedantic` will show you a couple small things to be fixed (these can be chars instead of strings, like `indent.contains(' ')`).

Reply:
Fixed!